### PR TITLE
Feature 2615 interview duration

### PR DESCRIPTION
**Pull request by 11karangm**
Targets #2615

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/HR/Routes/web.php [hunk 1]
```diff
@@ -55,6 +55,7 @@
             Route::get('Dailyapplicationcount', 'ReportsController@index')->name('recruitment.reports.index');
             Route::get('reportsCard', 'ReportsController@showReportCard')->name('recruitment.daily-applications-count');
             Route::get('rejected-applications', 'ReportsController@rejectedApplications')->name('recruitment.rejected-applications');
+            Route::get('applications-rounds-duration', 'ReportsController@ApplicationRoundsDuration')->name('recruitment.application-rounds');
```
Comment (GauravGusain98) [inline Modules/HR/Routes/web.php:58]: This way it is easier to understand what this route is doing. It is a good practice to keep the naming convention such that the name of the variable/route/controller, etc. defines its purpose
```suggestion
            Route::get('reports/applications-rounds-duration', 'ReportsController@ApplicationRoundsDuration')->name('recruitment.reports.application-rounds');
```

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Let's remove the button here, and add a title instead
image
Other than that, I have added a logic change in the issue description, because due to inconsistent data, its populating incorrect results. Let's change that.

File: resources/js/app.js [hunk 2]
```diff
@@ -197,6 +197,9 @@ $(document).ready(() => {
 	if ($("#myGraph").length) {
 		roundWiseRejectionsGraph();
 	}
+	if ($("#myGraphDuration").length) {
+		applicationRoundsDurationGraph();
+	}
```
Comment (GauravGusain98) [inline resources/js/app.js:202]: The name `myGraph` is sounding very unprofessional. It looks like that it is a dummy project. We should not use names like this. The one who has kept this name has done it incorrectly, can you please correct it.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Left some feedbacks. Will be reviewing it further once all the feedbacks till now have been implemented.

**Pull request closed without merging**